protos/health.proto

```
@@ -0,0 +1,112 @@
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package resq.health;
```

Buf (1.72.0)

[error] Files with package "resq.health" must be within a directory "resq/health" relative to root but were in directory "protos".

(PACKAGE_DIRECTORY_MATCH)